Better shape: Deprecate reshape, into_shape (#1310)
rust-ndarray/ndarray: README-quick-start.md

## Shape Manipulation

### Changing the shape of an array

> The shape of an array can be changed with `into_shape_with_order` method.
> (One can also use `to_shape` for this.)

Reviewer: ```suggestion
The shape of an array can be changed with the `into_shape_with_order` or `to_shape` method.
```
